File: container02_cgi/cgi-bin/confirm.pl
#!/usr/bin/perl
use CGI;
use strict;

my $q = CGI->new;
print $q->header;
print $q->start_html("Confirmado");

my $produto = $q->param("produto");
my $endereco = $q->param("endereco");

print "<h2>Pedido Confirmado!</h2>";
print "<p>Produto: $produto</p>";
print "<p>Endereço: $endereco</p>";
print "<a href='/index.html'>Voltar</a>";
print $q->end_html;

File: container05_php/html/confirm.php
<?php
session_start();
if (!isset($_SESSION["user"])) {
    header("Location: login.php");
    exit;
}
$db = new SQLite3('ecommerce.db');
$product_id = $_POST["product_id"] ?? null;
$product = $db->querySingle("SELECT nome FROM produtos WHERE id = " . intval($product_id));
?>
<h2>Pedido Confirmado!</h2>
<p>Produto: <?= htmlspecialchars($product) ?></p>
<p>Endereço: <?= htmlspecialchars($_POST["address"]) ?></p>
<p><a href="index.php">Voltar</a></p>
